NETProvider/src/FirebirdSql.Data.UnitTests/FbCommandTests.cs

```diff
@@ -730,6 +756,15 @@ public void NoCommandPlanTest()
 				Assert.DoesNotThrow(() => { plan=cmd.CommandPlan; });
 				Assert.IsEmpty(plan);
 			}
+			}
+			finally
+			{
+				using (var cmd = Connection.CreateCommand())
+				{
+					cmd.CommandText = "drop table NoCommandPlanTest";
```

Comment:
The `TearDown` would take care of that, no? IIRC you wanted to just cleanup the database to save time dropping and creating every time.